py/orbit/parsers/madx_parser.py

```diff
@@ -3,6 +3,8 @@
 import re
```

Comment:
I think for the issue b) I have a solution. I have to check how I can this include in the new parser. If I found a working solution I will open a new branch.

Reply:
Please don't create a new branch in the main repository. Instead follow the same way Dmitrii  submitted this PR: create your own fork first and then submit PR form your fork to the main repo.

Reply:
Ok.